### DISCUSSION TITLE
Why doesn't abseil Demangle print template types?

### DISCUSSION THREAD

**Discussion by MBkkt** [abseil/abseil-cpp#1687] [Q&A]
Maybe it's planned to be make this behavior optional?

**Comment by derekmauro** [comment]
I actually don't know why this was never implemented, but today by coincidence a contributor sent a change to add this functionality.

**Reply by MBkkt** [reply]
> but today by coincidence a contributor sent a change to add this functionality.

So it will be available soon? Nice :)
